webpack/js/modules.js

```diff
@@ -0,0 +1,47 @@
+module.exports = (ENV, { SRC }) => {
+  const eslintLoader = (ENV === 'production')
+    ? {
+      test: /\.jsx?$/,
+      enforce: 'pre',
+      loader: 'eslint-loader',
+      options: {
+        filePath: SRC,
+      },
+    }
+    : null;
+
+  return {
+    rules: [
+      eslintLoader,
+      {
+        // .js and .jsx files are caught
+        test: /\.jsx?$/,
+        exclude: /(node_modules|thirdparty)/,
+        loader: 'babel-loader',
+        options: {
+          presets: [
+            ['es2015', { modules: false }],
```

Comment:
Ah hah, I had used `es2015-webpack` actually, but then found that the later versions of babel supports it with what I have currently :D
Check out the readme for https://github.com/gajus/babel-preset-es2015-webpack